pkg/azurekeyvault/client/certificate_test.go

```diff
@@ -30,7 +30,7 @@ var (
 
```

Comment:
We should have a test for changing the chain order.

Reply:
I had a hard time figuring out how to create a erroneous certificate to use in the test. I'll have another look at it, maybe I can take a shortcut and still get the value from it.

Reply:
maybe if you create a chain in a .pem in the wrong order and then use:

`openssl pkcs12 -export -out mydomain.com.pfx -inkey mydomain.com.key -in combined.crt`

Reply:
Test added